frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -72,29 +73,16 @@ export function OngoingMissionCard({ mission }: MissionProps) {
                 </div>
             </TopContent>
             <BottomContent>
-                <VerticalContent>
-                    <Typography variant="meta" color="#6F6F6F">
-                        {'Status'}
-                    </Typography>
-                    <HorizontalContent>
-                        <MissionStatusDisplay status={mission.status} />
-                        <MissionProgressDisplay mission={mission} />
-                        <BatteryStatusView
-                            battery={mission.robot.batteryLevel}
-                            batteryStatus={BatteryStatus.Normal}
-                            robotStatus={mission.robot.status}
-                        />
-                    </HorizontalContent>
-                </VerticalContent>
-                <div>
-                    <Typography variant="meta" color="#6F6F6F">
-                        {'Robot'}
-                    </Typography>
-                    <Typography variant="body_short" color="#3D3D3D">
-                        {' '}
-                        {mission.robot.name}
-                    </Typography>
-                </div>
+                <HorizontalContent>
```

Comment:
BottomContent and HorizontalContent can be combined into one style. Don't need both